projects/1.21/assets/data-essence/datanessence/lang/zh_cn.json

```
@@ -0,0 +1,686 @@
+  "data_tablet.pages.essence_breaker.page2.text": " ◇ 可通过红石禁用。",
+  "data_tablet.pages.essence_furnace.flavor": " ◇ 铜质对流烧炼器",
+  "data_tablet.pages.essence_furnace.name": "源质熔炉",
+  "data_tablet.pages.essence_furnace.page1.text": "烧炼是一项常见加工手段，但普通熔炉太过低效和缓慢，不适合我们的大多数加工流程。我们很快发现让源质为加热元件注能的方法，这一方案也能烧炼物品，从而弥补了前文的两项缺点。",
+  "data_tablet.pages.essence_furnace.page2.text": " ◇ 工作速度比普通熔炉快25%。",
+  "data_tablet.pages.fluid_acquisition.flavor": " ◇ 100%吸收力",
+  "data_tablet.pages.fluid_acquisition.name": "流体的收集",
+  "data_tablet.pages.fluid_acquisition.page1.text": "为实际将流体收集并传输到机器里，可使用源质驱动泵，让泵收集其面朝方向的一桶流体。自然，可简单将此过程反转，从而制成倒出流体的同类机器。",
+  "data_tablet.pages.fluid_acquisition.page2.text": " ◇ 必须使用导向器选取“泵”面。",
+  "data_tablet.pages.fluid_bottler.name": "流体装瓶器",
+  "data_tablet.pages.fluid_bottler.page1.text": "某些时候需要将流体自动装入铁桶等容器。源质可以简化此过程。",
+  "data_tablet.pages.fluid_mixer.name": "流体混合器",
+  "data_tablet.pages.fluid_mixer.page1.text": "流体混合器是一台简单的机器：源质驱动发动机，达到一定转速后将物品和流体混合为均一稳定的新流体。我们已经发现了它在量产熔岩方面的用途。",
+  "data_tablet.pages.fluid_mixer.page2.text": " ◇ 必须插入装载数据的数据驱动器以指定合成配方。",
+  "data_tablet.pages.fluid_tank.flavor": " ◇ 未雨绸缪",
+  "data_tablet.pages.fluid_tank.name": "流体的存储",
+  "data_tablet.pages.fluid_tank.page1.text": "这件特制的储罐是在合理大小的空间内存储流体的简单方式，容量可观。可通过铁桶手动倒入。",
+  "data_tablet.pages.fluidworks.name": "流体的传输",
+  "data_tablet.pages.fluidworks.page1.text": "流体的操纵，尤其是在机器间输送流体，是我们工作中的重要流程。从对源质表现极好的已有设计出发，我们设计了流体节点和相应导线。它们的工作方式正如预期：能点对点传输任意流体。",
+  "data_tablet.pages.fluidworks.page2.text": " ◇ 节点的连接顺序决定了传输方向：先选中出发点，后选中到达点。\n ◇ 中转节点不会发出或接收。",
+  "data_tablet.pages.industrial_infuser_processing.flavor": " ◇ 为了好玩往物品里灌能量，还整出了些有用的东西",
+  "data_tablet.pages.industrial_infuser_processing.name": "工业灌注器加工",
+  "data_tablet.pages.industrial_infuser_processing.page1.text": "我们已在探索灌注器的实际应用。我们首先研究了最简单、最基础的工业源质。到目前为止，我们观测到下方列出的物品在暴露于工业源质时发生了多种有用的反应。有些成果些微方便了加工，还有些能从常见物品里产出各类必需资源。",
+  "data_tablet.pages.infuser.flavor": " ◇ 没那么精细的加工",
+  "data_tablet.pages.infuser.name": "灌注器",
+  "data_tablet.pages.infuser.page1.text": "构装器是要求精细加工的配方的理想工作台，但某些时候只需要把源质灌到物品里，无需追求精度。为此我们开发了灌注器。用法相当简单：用数据驱动器选择所需配方，放入所需源质和原材料，静待源质流入原材料。\n\n此机器的一种简单用途是将工业源质灌注入腐朽的材料，以将其硬化成某种皮革。",
+  "data_tablet.pages.infuser.page2.text": " ◇ 必须插入装载数据的数据驱动器以指定合成配方。",
+  "data_tablet.pages.infuser_lunar.name": "灌注月辉源质",
+  "data_tablet.pages.infuser_lunar.page1.text": "掌握月辉源质后，我们即尝试将它灌注入多种材料。产物很有意思：有些把平凡的物质变为它们的稀有变种，有些则成了获取特定资源的新方法。",
+  "data_tablet.pages.item_filter.flavor": " ◇ 不是钻石管道",
+  "data_tablet.pages.item_filter.name": "为物品分类",
+  "data_tablet.pages.item_filter.page1.text": "物品节点虽然极擅长传输物品，但在过滤和分类方面有缺陷。因此，处理工厂的产物经常会很……麻烦。我们制造了一种六面有颜色标记的盒子，同时在内部安装了逻辑单元，以按照所设过滤器为物品分类。这就是解决方案了。",
+  "data_tablet.pages.item_filter.page2.text": " ◇ 输出面必须存在容器以供存储。\n ◇ 只接受有匹配过滤器的物品。\n ◇ 经过滤的面不会接受输入。",
+  "data_tablet.pages.itemworks.name": "物品的传输",
+  "data_tablet.pages.itemworks.page1.text": "物品的传输和寻路大概是源质驱动工厂的运转中最重要的事项了，仅次于源质本身的传输。为达成此目标，我们从已有设计出发制造了物品节点和相应导线。它们能将物品传输到我们指定的位置。",

```

```suggestion
  "data_tablet.pages.itemworks.page1.text": "仅次于源质本身的传输，物品的传输和寻路大概是源质驱动工厂的运转中第二重要的事项了。为达成此目标，我们从已有设计出发制造了物品节点和相应导线。它们能将物品传输到我们指定的位置。",
```
~~都次于源质传输了还是最重要的事项吗（~~